group-generators/generators/wiw-crypto-memento/index.ts

```diff
@@ -0,0 +1,44 @@
+// sismo-hub/group-generators/generators/wiw-crypto-memento/index.ts
+
+import fs from 'fs';
+import path from 'path';
+import {
+    Tags,
+    ValueType,
+    GroupWithData,
+    AccountSource,
+} from "topics/group";
+import {
+    GenerationContext,
+    GenerationFrequency,
+    GroupGenerator,
+} from "topics/group-generator";
+
+// Here you are hacker 😈
```

Comment:
remove the commentary from the tutorial (same remark for the other group generator)

Reply:
Done.